src/components/MessageBoard/Message.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useMemo } from 'react'
+import { IMessage } from '../../types';
+import { emojify } from 'node-emoji';
+import ChatMsg from '@mui-treasury/components/chatMsg/ChatMsg'
+
+export interface IMessageProps {
+  message: IMessage;
+  name: string;
+}
+
+export function Message({ message: { owner, text }, name }: IMessageProps) {
+  const trimmedName = useMemo(() => name.trim().toLowerCase(), [name]);
```

Comment:
Lately I've been using the methods: `useMemo` and `useCallback` more and more. The real benefit of it can be seen here (this line).

**In the previous version:**
Every time the component was rendered, the name would need to be trimmed and evaluated. While the performance hit for this is very minimal, it's still a process that is happening.

**In this version:**
The `trimmedName` value will only be reprocessed when the dependency `name` changes. This means on subsequent renders this is treated as a static var, boosting performance.

The same performance logic happens with the `useCallback` hook, just that one is for functions whereas `useMemo` is for variables. The trigger for when they re-compute their exported values is the same as `useEffect`.